middlewares/authn.sql

```diff
@@ -23,22 +23,8 @@ INSERT INTO enrollments
         course_instances AS ci
         JOIN pl_courses AS c ON (c.id = ci.course_id)
     WHERE
-        u.uid = $uid
-        AND c.short_name = $course_short_name
+        u.user_id = $user_id
+        AND c.example_course IS TRUE
```

Comment:
This feels safer than relying on a course name?

Reply:
Yep, although it has the downside that it will enroll into all example courses if there is more than one (there is only one at the moment). I don't think this is a problem.

Reply:
And of course the `IS TRUE` is not needed, but it's not doing any harm.